docs/_includes/social.html

```diff
@@ -0,0 +1,14 @@
+<ul class="social-media-list">
```

Comment:
What are these for? Do we want this?

Reply:
Same for the above individual social icons.

Reply:
@Widcket, yes. To override **one** include, we need to respect the entire folder structure here https://github.com/jekyll/minima/tree/master/_includes. It is a common practice with github themes until you have your own. I tried adding only the header to our _includes folder and the `make start-doc` process fails.